CalculatorAppJMS/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,233 @@
+        writer.WriteStartObject();
+        writer.WritePropertyName("Operations");
+        writer.WriteStartArray();
+    }
+
+    public double DoOperation(double num1A, double num2A, string operation)
+    {
+        string operationA = "";
+        double resultA = double.NaN;
+        //JSON
+        writer.WriteStartObject();
+        writer.WritePropertyName("Operand1");
+        writer.WriteValue(num1A);
+        writer.WritePropertyName("Operand2");
+        writer.WriteValue(num2A);
+        writer.WritePropertyName("Operation");
+
+        switch (operation)
+        {
+            case "+":
+                resultA = num1A + num2A;
+                operationA = "+";
+                writer.WriteValue("Add");
+                break;
+            case "-":
+                resultA = num1A - num2A;
+                operationA = "-";
+                writer.WriteValue("Subtract");
+                break;
+            case "*":
+                resultA = num1A * num2A;
+                operationA = "*";
+                writer.WriteValue("Multiply");
+                break;
+            case "/":
+                resultA = num1A / num2A;
+                operationA = "/";
+                writer.WriteValue("Divide");
+                break;
+            case "s":
+                resultA = Math.Sqrt(num1A);
+                operationA = "SquareRoot";
+                writer.WriteValue("SquareRoot");
+                break;
+            case "p":
+                resultA = num1A;
+                for(int i = 1; i < num2A; i++)
+                { 
+                    resultA *= num1A;
+                }
+                operationA = "powerOf";
+                writer.WriteValue("Power of");
+                break;
+            case "i":
+                resultA = Math.Sin(num1A * Math.PI / 180);
+                operationA = "Sin";
+                writer.WriteValue("Sin");
+                break;
+            case "o":
+                resultA = Math.Cos(num1A * Math.PI / 180);
+                operationA = "Cos";
+                writer.WriteValue("Cos");
+                break;
+            case "y":
+                resultA = Math.Tan(num1A * Math.PI / 180);
+                writer.WriteValue("Tan");
+                break;
+            case "m":
+                resultA = num1A * 10;
+                operationA = "Tan";
+                writer.WriteValue("Multiply by 10");
+                break;
+            default:
+                break;
+        }
+        //JSON
+        writer.WritePropertyName("Result");
+        writer.WriteValue(resultA);
+        writer.WriteEndObject();
+
+        OperationRecord operationRecord = new OperationRecord
+        {
+            Id = operationId,
+            numA1 = num1A,
+            operation = operationA,
+            NumA2 = num2A,
+            resultA = resultA
+        };
+        operationsList.Add(operationRecord); //adds the operation record to the operationsList that we are using

```

Comment:
Maybe this comes from Microsoft's tutorial, but in general refrain from commenting when the code is self-explanatory, such as in line 111.

Reply:
Deleted! Those were comments to help me understand or guide me. Forgot to delete that one.